resources/help.txt

```diff
@@ -245,6 +248,22 @@ Options:
 
 --[/]--
 
+--[init|website]--
+
+Usage:
+    $ appbuilder init website [--appid <App ID>]
+
+Initializes an existing Mobile Website project for development in the current directory. If the folder contains an existing AppBuilder project (created with
```

Comment:
Initializes an existing Mobile Website project for development in the current directory. If the dorectory contains an existing AppBuilder project synchronized from GitHub, the project retains the existing project configuration.

-> We don't have mobile website projects in VSE.
-> Apply Toshko's directory comment to the rest of the init commands.